pkg/operator/handlers/dapr_handler.go

```diff
@@ -26,12 +25,12 @@ const (
 
 // DaprHandler handles the lifetime for Dapr CRDs
 type DaprHandler struct {
-	client          scheme.Interface
+	client          *kubernetes.Clients
```

Comment:
The struct name is plural, `Clients`, but assigned into a singular form field, `client`.
Why not change the field name to `clients`?

However, if changed to `clients`, its a little weird having methods like `ServiceExists` on a field called `clients`. Its not clear this way which one of the clients is actually used to perform the operation.